### ISSUE TITLE
STL reader is a stub

### ISSUE THREAD

**Issue by Kulitorum** [jlerouge/glanc3d#2]
Your STL reader and writer is very incomplete and makes a lot of assumptions about the file, you should use a library instead.

**Comment by jlerouge** [comment]
Yes, probably. It's an old an unmaintained project though. Are you looking for a mesh viewer ?

**Comment by Kulitorum** [comment]
Well, I am looking for something that I can use as a base for a slicer for my concrete 3D printer :)
https://www.youtube.com/user/KulitorumCom/videos
